backend/curriculum_tracking/serializers.py

```diff
@@ -928,3 +956,12 @@ def get_reviewer_user_emails(self, instance):
     def get_status(self, instance):
         if instance.agile_card:
             return instance.agile_card.status
+
+    def get_number_of_times_card_moved_to_review_feedback(self, instance):
+        number_of_times_card_moved_to_review_feedback = len(
+            LogEntry.objects.filter(effected_user=instance.recruit_users.first(),
+            object_1_id=instance.pk, 
+            event_type=EventType.objects.get(name='CARD_MOVED_TO_REVIEW_FEEDBACK')),
```

Comment:
This query results in multiple SQL queries being fired off. We could mke this more efficient by doing something like:

```
filter(event_type__name=CARD_MOVED_TO_REVIEW_FEEDBACK)
```